index.html

```diff
@@ -269,47 +318,103 @@ <h4>SvgRenderingTemplate</h4>
             <tr>
               <td>id</td>
               <td>
-A URL that dereferences to an SVG image [[SVG]] with an associated
-`image/svg+xml` media type.
+An OPTIONAL [=string=] that follows the [[[URL]]] and, when fetched,
+dereferences to a render template.
               </td>
             </tr>
             <tr>
               <td>type</td>
               <td>
-The <code>type</code> property MUST be <code>SvgRenderingTemplate</code>.
+A REQUIRED [=string=] that MUST be the value `TemplateRenderMethod`.
+              </td>
+            </tr>
+            <tr>
+              <td>renderSuite</td>
+              <td>
+A REQUIRED [=string=] that identifies the algorithms that are used for
+generating the concrete rendering.
               </td>
             </tr>
             <tr>
               <td>name</td>
               <td>
-A name that can be used by a display to express the type of rendering that
-is associated with the render method. This property might be used in a
+An OPTIONAL human-readable [=string=] that can be displayed to provide a hint to
+the type of rendering that will be performed. This property might be used in a
 graphical interface that enables an individual to select between multiple
 presentation modes.
               </td>
             </tr>
             <tr>
-              <td>css3MediaQuery</td>
+              <td>description</td>
               <td>
-A CSS Level 3 Media Query that can be used to specify desired characteristics
-in the display device. If there are multiple render methods, the best match
-to the CSS Level 3 Media Query will be used by default. It is expected that
-an individual could override this preference through a manual process.
+An OPTIONAL human-readable [=string=] that provides a more involved description
+than `name` of when the particular rendering might be useful.
+              </td>
+            </tr>
+            <tr>
+              <td>renderProperty</td>
```

Comment:
This seems to me like a property that could potentially misguide the wallet from a) honest oversight (issuer mistakenly omits potentially exposed properties) or b) intentional deceit (issuer intentionally omits potentially exposed properties). Couldn't/shouldn't this exposure instead be calculated deterministically by the wallet using the credential and template data?

Reply:
It's not always possible for the wallet to do that calculation. If it is possible, it should be done that way, but we can't assume that for some purely binary blobs like NFC payloads. Those require issuer trust -- which isn't that different from trusting the issuer not to put an SSN in a "name" field.

Reply:
This property makes sense _if it's required_ so it would become the foundation for populating the data passed into the render method.